packages/create-vite/src/index.ts

```diff
@@ -593,10 +593,13 @@ function pkgFromUserAgent(userAgent: string | undefined): PkgInfo | undefined {
 }
 
 function setupReactSwc(root: string, isTs: boolean) {
+  // renovate: datasource=npm depName=@vitejs/plugin-react-swc
+  const reactSwcPluginVersion = '3.7.2'
```

Comment:
```suggestion
  const reactSwcPluginVersion = '3.7.0'
```
I'm not sure how to test this PR. Maybe setting a lower version so that renovate tries to update it?

Reply:
I'm ok with not testing this for now. Verifying the regex manually it seems that it should work as expected though.